packages/fury-adapter-oas3-parser/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Fury OAS3 Parser Changelog
 
+## Master
+
+### Enhancements
+
+- Adds support for the `info.termsOfService` schema extension.
```

Comment:
I'd remove the wording regarding schema and extensions this isn't part of schema object or an extension. Perhaps something along the lines of:

```suggestion
- Adds support for `termsOfService` in 'Info Object'.
```

There's also a file called `STATUS.md` in this directory which tracks what features are supported. Could you take a look at that file and update `termsOfService`.

Reply:
The STATUS file is at https://github.com/apiaryio/api-elements.js/blob/ed99d4967eb71383e9328cb3c56dcbd74e43262a/packages/fury-adapter-oas3-parser/STATUS.md#info-object

Reply:
ok, change on next commit